hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3InitiateMultipartUploadRequest.java

```diff
@@ -158,9 +158,10 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
           .setOmKeyLocationInfos(Collections.singletonList(
               new OmKeyLocationInfoGroup(0, new ArrayList<>())))
           .setAcls(OzoneAclUtil.fromProtobuf(keyArgs.getAclsList()))
+          .setObjectID(transactionLogIndex)
```

Comment:
Missing to add objectID and updateID in multipartKeyInfo

Reply:
Thanks for catching this. Updated the patch.